crates/compiler/solve/tests/solve_expr.rs

```diff
@@ -1469,6 +1469,34 @@ mod solve_expr {
         infer_eq("{ x: 5, y : 3.14 }.x", "Num *");
     }
 
+    #[test]
+    fn record_literal_accessor_function() {
+        infer_eq(".x { x: 5, y : 3.14 }", "Num *");
+    }
+
+    #[test]
+    fn tuple_literal_accessor() {
+        infer_eq("(5, 3.14 ).0", "Num *");
+    }
+
+    #[test]
+    fn tuple_literal_accessor_function() {
+        infer_eq(".0 (5, 3.14 )", "Num *");
+    }
+
+    #[test]
+    fn tuple_literal_ty() {
+        infer_eq("(5, 3.14 )", "( Num *, Float * )*");
+    }
+
+    #[test]
+    fn tuple_literal_accessor_ty() {
+        infer_eq(".0", "( a )* -> a");
+        infer_eq(".4", "( _, _, _, _, a )* -> a");
+        infer_eq(".5", "( ... 5 omitted, a )* -> a");
+        infer_eq(".200", "( ... 200 omitted, a )* -> a");
+    }
```

Comment:
Doesn't need to happen now, but in a follow up can we add a test to ensure that tuple accessors are in fact generalized? That is, I'm thinking of a test like

```
get0 = .0

{ a: get0 (1, 2), b: get0 ("a", "b", "c") }
```

to make sure it doesn't error